data/korath/korath weapons.txt

```diff
@@ -577,6 +737,163 @@ effect "slicer impact"
 	"random velocity" .1
 	"velocity scale" -.002
 
+
+outfit "Digger Mining Turret"
+	category "Turrets"
+	cost 509000
+	thumbnail "outfit/korath digger turret"
+	"mass" 14
+	"outfit space" -16
+	"weapon capacity" -16
+	"asteroid scan power" 15
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/digger"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/digger turret"
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 1.5
+		"turret turn" 4.6
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 2.9
+		"firing heat" 4.9
+		"shield damage" 0.8
+		"hull damage" 1.3
+		"minable damage" 5.2
+		"disruption damage" .08
+	description `When the Korath Exiles needed a cheap mining laser for their large vessels, they re-engineered this ancient but robust design. The mild shield disruption effect makes it somewhat useful in combat if required.`
+
+outfit "Digger Mining Beam"
+	category "Guns"
+	cost 419000
+	thumbnail "outfit/korath digger"
+	"mass" 10
+	"outfit space" -10
+	"weapon capacity" -10
+	"asteroid scan power" 10
+	"gun ports" -1
+	weapon
+		sprite "projectile/digger thin"
+			"frame rate" 20
+		sound "korath digger"
+		"hit effect" "korath digger hit"
+		"inaccuracy" 0.1
+		"velocity" 480
+		"lifetime" 1
+		"reload" 1
+		"firing energy"  2.1
+		"firing heat" 4.3
+		"shield damage" 0.5
+		"hull damage" 0.9
+		"minable damage" 4.6
+		"disruption damage" .06
+	description `The gun version of the classic Korath mining laser allows a larger mothership to release a cloud of fighter-sized miners to harvest resources when working faster-moving or more dangerous asteroid fields.`
+
+effect "korath digger hit"
+	sprite "effect/korath digger hit"
+		"no repeat"
+		"frame rate" 30
+	"lifetime" 5
+	"random angle" 30
+	"velocity scale" .005
+
+
+outfit "Langrage Hyper-Heaver"
+	category "Secondary Weapons"
+	cost 3450000
+	thumbnail "outfit/korath heaver"
+	"mass" 46
+	"outfit space" -46
+	"weapon capacity" -46
+	"fuel capacity" 200
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/heaver hyperspace"
+			"frame rate" 12
+			rewind
+		"hardpoint sprite" "hardpoint/korath heaver"
+		icon "icon/korath heaver"
+		velocity 30
+		lifetime 8
+		"hit force" 3600
+		"hit effect" "heaver hyperspace hit"
+		sound "heaver"
+		submunition "Heaver Shard" 20
+		"turret turn" 2.2
+		"reload" 40
+		"burst reload" 12
+		"burst count" 6
+		"firing energy" 31
+		"firing heat" 190
+		"firing fuel" 12.5
+		"inaccuracy" 0.5
+	description `Langrage Hyper-Heavers raise the use of improvised ammunition to a fine art. A small Banisher propels scrap metal, nearby asteroids, compressed garbage, or other debris into an unstable hyperspace window.`
+	description `At close range, the jumping langrage merely forces the enemy back. Further along, being forced to hyper transforms it into a rapidly-expanding cloud of deadly projectiles as it returns to normal space.`
```

Comment:
Maybe instead of hyperspace, maybe the flavour text could just be "a combination of banisher parts, heat lances and afterburner plumbing has resulted in this horrifying yet effective weapon that turns scrap waste into a chaotic spray of projectiles."...or something along those lines.

Reply:
Humanity has a very *long* tradition of declaring stuff impossible, then figuring out how to do it later anyway. I see no reason why a human hyperspace engineer should have any real capability to dictate the limitations on how a much more advanced alien race uses technology.

It might be a bit of an obstinate attitude, but I'd actually consider the fact that human in-universe people declare it to be impossible to make it even *more* appropriate that the Korath actually manage to do so. 

Personally, I suspect that those hyperspace engineers do not, in fact, know that hyperspace weaponry is impossible - but rather they are trying desperately to keep military attention focused away from hyperspace so that they can keep their field for peaceful purposes. Perhaps, drawing inspiration from history, they are like the atomic scientists, except they have realized that their field may indeed be turned to warfare, and are trying pre-emptively to convince everyone that it cannot. 

As such, I'd personally like to keep this lore. I like the idea of short range (aka in-system) military applications of hyperspace technology, and the Korath are unquestionably the best positioned race to have developed it. (given that they are basically second only to Drak and Pug in their understanding of hyperspace related technology)

Reply:
> Humanity has a very long tradition of declaring stuff impossible

Case in Point:
The New York Times, on Oct 9th, 1903: "[...] it might be assumed that the flying machine which will really fly might be evolved by the combined and continuous efforts of mathematicians and mechanicians in from one million to ten million years."

The Wright brothers first flew on December 17, 1903. (two months and 8 days later)

Reply:
One of the themes I've been trying to create, and that you'll see more of in Stage II, is that while the Efreti developed Korath gravity-manipulation tech for peaceful purposes, the Exiles have turned it to... well, I was about to say war, but I suppose it might more aptly be described as *crime*, for the most part.

Hyperdrive technology is suggested to be related to or reliant upon gravitics, and Korath have gone beyond that to master enough of the warp and weft to create jump drives, so I think it would be reasonable and in-theme for the Exiles, and possibly *only* the Exiles until we get other aggressive T2 or higher factions, to have hyper-based weaponry.

But, I get that it's unusual and is lore-establishing for the whole setting, not just the Korath.  So, I wouldn't be... too opposed to re-loring this as a strictly *gravity*-based weapon.

I think we'd need to swap out the sounds and the effects for the first stage if we did that, though.  The animation and its effect I think are great for showing what happens when you shove a container of garbage through a small, temporary, unstable hyperlink in your opponent's direction, but wouldn't make sense if this is just a big 'ol banisher being used to throw rice.  Nor would the fuel usage, which is a major balancing point.

Reply:
I think just making it directly grav manipulation based still works, but I agree that we should reconsider fuel cost and some of the effects then.

But considering it's literally a way to weaponize otherwise worthless debris and, essentially, trash, I think it should be something they start with. It really feels like an early improvisation after the Exile.

Later, beefier versions made to throw large solid chunks that can fill the role of traditional orbital bombardment that Zitchas also wants would be much more suited to that.